Java V2 Added FSA content (#5347)
awsdocs/aws-doc-sdk-examples: javav2/usecases/creating_fsa_app/README.md

### Update the Lambda functions

After you execute the AWS CDK script, the Lambda functions are created. However, you need to replace the ones installed using the AWS CDK with the ones you created by using the AWS SDK for Java. To perform this task, you can run Java code that automatically updates the Lambda functions using a FAT JAR. Perform the following tasks:

1. Update the FSAApplicationResources with the AWS resource names that the AWS CDK script created. Make sure to update the **STORAGE_BUCKET** value. 

2. Create the FAT JAR by running **mvn package**. The JAR file can be located in the target folder.

3. Use the AWS Management Console to place the FAT JAR into the Amazon S3 bucket named **STORAGE_BUCKET**. (This was created by the AWS CDK script.)  

4. Run the **ModifyLambda** Java class.  Make sure to specify the exact AWS Lambda names, which are provided in the AWS Lambda console. Otherwise, the code does not work. When done, you see a message that the AWS Lambda functions have been updated with the FAT JAR. 

### Update the permission that the Lambda roles use

> Each Lambda function has a role used to invoke the Lambda function. For example, the **fnAnalyzeSentiment** function has this role.

Reviewer: Recommend putting the name of the role into this text to make it more accessible for screen readers. 
"For example, the **fnAnalyzeSentiment** function has the <INSERT_ROLE_NAME> role."

![AWS Photo Analyzer](images/role.png)

> You need to ensure each role has the correct service permission in order to invoke the corresponding AWS Service. If you do not modify the role's permission that each Lambda function uses, then you will encounter a permission error. 
> 
> **Note**: For information about how to modify the a role's permission, see [Using service-linked roles](https://docs.aws.amazon.com/IAM/latest/UserGuide/using-service-linked-roles.html).

Reviewer: ```suggestion
Make sure each role has the correct service permission to invoke the corresponding AWS service. If you do not modify the role's permission for each Lambda function, you will encounter a permission error. 

**Note**: For information about how to modify role permissions, see [Using service-linked roles](https://docs.aws.amazon.com/IAM/latest/UserGuide/using-service-linked-roles.html).
```

> #  Creating a Feedback Sentiment Analyzer application using the SDK for Java

Author: This follows the Java Template which all have similair topics. 

Prerequisites
Understand the Feedback Sentiment Analyzer application
Create an IntelliJ project
Add the POM dependencies to your project
Create the Java classes

Same for all Java use cases,
